DEVELOPMENT.md

```diff
@@ -98,7 +99,7 @@ To [build, deploy and run your Tekton Objects with `ko`](#install-pipeline), you
 
 1. `GOROOT`: Set `GOROOT` to the location of the Go installation you want `ko` to use for builds.
 
-    > **NOTE** You may need to set `GOROOT` if you installed Go tools to a a non-default location or have multiple Go versions installed.
+    > **Note**: You may need to set `GOROOT` if you installed Go tools to a a non-default location or have multiple Go versions installed.
```

Comment:
NIT: extra `a` here 🙃  `to a a non-default location`

Reply:
Fixed the "a a" that was from previous PR ;)